packages/mdc-snackbar/mdc-snackbar.scss

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2017 Google Inc.
+// Copyright 2018 Google Inc.
```

Comment:
We typically don't change the copyright year in pre-existing files. I can buy the argument in other cases that the files are effectively being replaced, but this one especially seems somewhat intact.

Reply:
I actually rewrote this file from scratch, so I figured that needed a year bump